docs/docs/advanced-use/transient-traits.md

```diff
@@ -0,0 +1,585 @@
+---
+title: Transient Traits and Identities
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
+
+By default, Flagsmith [stores all Traits](/basic-features/managing-identities#identity-and-trait-storage) associated
+with an Identity to evaluate flags. This default behavior works for most use cases. However, there are scenarios where
+you may want more control over which Traits or Identities are stored. Using the Flagsmith API and SDKs, you can mark
+Traits and Identities as transient — meaning they are used for flag evaluation but are not stored.
+
+## Overview and Use Cases
+
+Transient Identities and Traits are particularly useful when you need to run flag evaluations without storing specific
+data in Flagsmith. Below are common scenarios where transient Traits or Identities come in handy.
+
+### Anonymous Identities
```

Comment:
I feel like it would be better to start with "Skipping Personal Identifiable Information" as it doesn't introduce the blank identifier but just a blank trait, which is simpler to understand.

Reply:
Agreed. Changed the order.